.github/workflows/ci.yml

```diff
@@ -21,6 +21,6 @@ jobs:
       - name: Run Linter
         run: npm run lint
       - name: Run Tests
-        run: npm test
+        run: npm run test
```

Comment:
Nice work on the tests. I am note sure which frontend and backend instance the test is running against. Are these instances available for Github actions?